airflow_home/dags/scripts/citytest_sf_appointments_etl.py

```diff
@@ -0,0 +1,114 @@
+""" OOC Proposed Retail script """
```

Comment:
ignore this file for now plz. it's not up and working and not part of launch today